src/index.js

```diff
@@ -12,17 +12,17 @@ const DEFAULT_EXTS = [ '.mjs', '.js', '.json', '.node' ];
 const readFileAsync = file => new Promise((fulfil, reject) => fs.readFile(file, (err, contents) => err ? reject(err) : fulfil(contents)));
 const statAsync = file => new Promise((fulfil, reject) => fs.stat(file, (err, contents) => err ? reject(err) : fulfil(contents)));
 const cache = fn => {
-	let cache = Object.create(null);
-	const wrapped = (s, cb = false) => {
-		if (s in cache === false) {
+	const cache = new Map();
+	const wrapped = (s, cb) => {
+		if (cache.has(s) === false) {
 			cache[s] = fn(s).catch(err => {
```

Comment:
The correct operation is `cache.set(s)`

Reply:
Also I keep wondering that I have not the slightest idea what `s` might be. Would be nice to use a proper variable name here while we are at it.